.github/workflows/stale.yml

```
@@ -0,0 +1,33 @@
+name: Stale Check
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: '0 0 * * *'
+permissions:
+  issues: write
+  pull-requests: write
+jobs:
+  stale:
+    # only run on repos in prometheus org, and skip this workflow in forks.
+    if: github.repository_owner == 'prometheus'
```

This won't work with `prometheus-community`.

```suggestion
    if: github.repository_owner == 'prometheus' || github.repository_owner == 'prometheus-community' # Don't run this workflow on forks.
```